assets/locales/en.json

```diff
@@ -105,5 +105,10 @@
     "title": "Port is busy",
     "message": "The port { port } is not available. Do you want to use { alt } instead?",
     "action": "Use port { alt } instead"
+  },
+  "multipleBusyPortsDialog": {
+    "title": "Ports are busy",
+    "message": "Some of the API or Gateway addresses cannot be binded to because ports are busy.",
+    "action": "I will fix the configuration"
```

Comment:
Suggestion: Open the config file in my editor?

Reply:
Close & Open Config File sounds good?